mathesar_db/tests/conftest.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
can we call this `base.py` since we're using `base.py` for similar setup files in other places?

Reply:
Naming it `conftest.py` gives us some `pytest` magic that automatically makes the fixtures defined in that file available to every test.  We'll lose that if we rename it.

Reply:
ah okay, I didn't realize it was a special `pytest` name, that's fine.